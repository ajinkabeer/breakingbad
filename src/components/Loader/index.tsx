import './style.css';

const Loader = () => (
  <div className="container">
    <p>Loading..</p>
  </div>
);

export default Loader;
