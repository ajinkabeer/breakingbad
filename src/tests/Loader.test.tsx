import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Loader from '../components/Loader';

describe('Header Component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('renders the correct component', () => {
    const { getByText } = render(<Loader />);
    getByText('Loading..');
  });
});
