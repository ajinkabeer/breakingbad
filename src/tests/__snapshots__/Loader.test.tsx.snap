// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <p>
      Loading..
    </p>
  </div>
</DocumentFragment>
`;
